apps/services/auth/public-api/test/setup.ts

```diff
@@ -154,6 +155,10 @@ export const setupWithAuth = async ({
         .useValue(createMockEinstaklingurApi(nationalRegistryUser))
         .overrideProvider(RskRelationshipsClient)
         .useValue(RskProcuringClientMock)
+        .overrideProvider(CompanyRegistryClientService)
+        .useValue({
+          getCompany: jest.fn().mockResolvedValue({}),
+        })
```

Comment:
I'm not seeing why this is a necessary change in this PR 🤷

Reply:
True, this change is not needed for this PR, reverted. We should address this at some point though since currently the tests attempt to call the actual api.